### PR TITLE
Allow partners carousel on mobile

### PR DESCRIPTION
**Pull request by annebelleo**
## Description

-Allows installation for MacOS devices
-changed home page to allow carousel to work on mobile for the partners

Fixes #

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have added tests to cover my changes.
- [ ] All new code is fully covered by tests (see coverage report)

### CODE REVIEW

File: secret_settings.py [hunk 1]
```diff
@@ -21,5 +21,11 @@
             secrets = json.load(f)
     else:
         raise FileNotFoundError("secrets.json file not found. Please refer to the Cosmos website wiki.")
+elif sys.platform.startswith("darwin"):
+    if os.path.exists("/etc/secrets.json"):
+        with open("/etc/secrets.json", "r") as f:
+            secrets = json.load(f)
+    else:
+        raise FileNotFoundError("secrets.json file not found. Please refer to the Cosmos website wiki.")
```
Comment (bcvandendool) [inline secret_settings.py:29]: There is no difference in the file handling for macos compared to linux it seems, so it is cleaner to just change the if statement on line 9 to also hold for macos

File: apps/core/static/cosmos/css/about.css [hunk 2]
```diff
@@ -44,4 +44,11 @@
...
-}
\ No newline at end of file
+}
+
+span.carousel-control-next-icon {
+  background-image: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' fill='%23ff0000' width='8' height='8' viewBox='0 0 8 8'%3e%3cpath d='M2.75 0l-1.5 1.5L3.75 4l-2.5 2.5L2.75 8l4-4-4-4z'/%3e%3c/svg%3e");
+}
+
+span.carousel-control-prev-icon {
+  background-image: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' fill='%23ff0000' width='8' height='8' viewBox='0 0 8 8'%3e%3cpath d='M5.25 0l-4 4 4 4 1.5-1.5L4.25 4l2.5-2.5L5.25 0z'/%3e%3c/svg%3e");
```
Comment (bcvandendool) [inline apps/core/static/cosmos/css/about.css:54]: Is there a reason for changing these icons? They now are kinda obnoxiously red, and always visible even when not needed. Controls were never really intended for this carousel, as the idea was to have it automatically cycle through them if there is not enough space to fit all of them.

**Review by bcvandendool — CHANGES_REQUESTED** [review]
Can you change back the formatting to how it was originally? It makes it easier to review, and keeps to the formatting used in the rest of the project.

File: apps/core/static/cosmos/css/about.css (hunk 2 above)
Comment (annebelleo) [reply apps/core/static/cosmos/css/about.css:54]: Ah, the red wasn't showing for me so I forgot to remove these lines
Why no controls though? If someone with a short attention span is scrolling through and only sees one sponsor they miss the other one(s), while with controls it's clear that there are multiple and will still work even if there is only one

File: secret_settings.py (hunk 1 above)
Comment (annebelleo) [reply secret_settings.py:29]: will do!

File: apps/core/static/cosmos/css/about.css (hunk 2 above)
Comment (bcvandendool) [reply apps/core/static/cosmos/css/about.css:54]: I guess that it makes sense for mobile yeah, not sure if it is needed for desktop though, as that should show 3 at the same time. But it is fine to add it either way, just less obtrusive as it is not the main thing we want to focus attention to.

**Pull request closed without merging**